client/modules/IDE/components/Editor/index.jsx

```diff
@@ -75,7 +80,7 @@ window.HTMLHint = HTMLHint;
 
 const INDENTATION_AMOUNT = 2;
 
-class Editor extends React.Component {
+class EditorV2 extends React.Component {
```

Comment:
Let's rename this back to `Editor` before merging.

Reply:
got it! done, check it out @lindapaiste